api/bnd.bnd

```diff
@@ -1,12 +1,26 @@
 -exportcontents: \
     org.eclipse.microprofile.*
 
-Import-Package: \
-    javax.enterprise.util;version="[1.1,3)", \
-    *
-
 Bundle-SymbolicName: org.eclipse.microprofile.config
 
 # For reproducible builds
 -noextraheaders: true
 -snapshot: SNAPSHOT
+
+-contract: *;effective:=active
+-define-contract: \
+    osgi.contract;\
+        osgi.contract=JavaCDI;\
+        version:List<Version>='2.0,1.1';\
+        uses:='\
+            javax.decorator,\
+            javax.enterprise.context,\
+            javax.enterprise.context.spi,\
+            javax.enterprise.event,\
+            javax.enterprise.inject,\
+            javax.enterprise.inject.spi,\
+            javax.enterprise.util',\
+    osgi.contract;\
+        osgi.contract=JavaInject;\
+        version:Version='1.0';\
```

Comment:
This contract is not in the generated manifest? (when built on my local system).

Does is matter that `version` is not `List<Version>`?

Reply:
So, this is was a weird issue. I tried in both forms List and scalar, but neither caused the contract to be added. Thoughts? I suspect the issue is because the only reference is as a meta-annotation. Thoughts?